apps/server/priv/boot.exs

```diff
@@ -0,0 +1,17 @@
+[script_dir] = System.argv()
```

Comment:
maybe we just put this file in `bin` at the top level and add that to the code path?